src/org/omegat/Main.java

```diff
@@ -255,6 +254,11 @@ private static void applyConfigFile(String path) {
      * Execute standard GUI.
      */
     protected static int runGUI() {
+        ClassLoader cl = ClassLoader.getSystemClassLoader();
+        MainClassLoader mainClassLoader = (cl instanceof MainClassLoader)? (MainClassLoader) cl: new MainClassLoader(cl);
```

Comment:
Project whitespace conventions

```suggestion
        MainClassLoader mainClassLoader = (cl instanceof MainClassLoader) ? (MainClassLoader) cl : new MainClassLoader(cl);
```